core/api/src/main/java/com/blazebit/persistence/spi/DbmsDialect.java

```diff
@@ -205,6 +206,22 @@
     public boolean supportsJoinsInRecursiveCte();
 
     /**
+     * Returns true if the dbms supports row value constructor syntax, false otherwise.
+     *
+     * @return Whether row value constructor syntax is supported by the dbms
+     * @since 1.2.0
+     */
+    public boolean supportsRowValueConstructor();
+
+    /**
+     * Returns true if the dbms supports <, <=, >, >=, =, <> comparison operations for row values.
```

Comment:
I just meant that the documentation suggests `=`and `<>` are only supported if this method returns true. Anyways, I think you got the gist.